jetty-core/jetty-deploy/src/test/java/org/eclipse/jetty/deploy/DeploymentScannerTest.java

```diff
@@ -370,23 +371,37 @@ public void testActionListXmlAndWarWithXmlRemoved() throws IOException
         deploymentScanner.pathsChanged(changeSet);
     }
 
+    @Test
+    public void testDefaultWeights()
+    {
+        DeploymentScanner deploymentScanner = new DeploymentScanner(new Server());
+        assertThat(deploymentScanner.getDefaultWeight("core"), is(200));
```

Comment:
There is too much magic here.
Where the 200 come from?

I would expect this test to call `configureEnvironment("core", 200)` before the asserts, so it's clear where the 200 comes from.